test/fixtures/app.js

```diff
@@ -6,6 +6,10 @@
 var loopback = require('loopback');
 var app = loopback();
 
+app.dataSource('db', { connector: 'memory' });
+var User = loopback.createModel('User');
+app.model(User, { dataSource: 'db' });
+
```

Comment:
I had to add something to the app template so my stdout buffer is long enough to have this error